aws_kms_pkcs11.cpp

```diff
@@ -486,6 +515,21 @@ CK_RV C_FindObjectsFinal(CK_SESSION_HANDLE hSession) {
     return CKR_OK;
 }
 
+CK_RV C_CopyObject(CK_SESSION_HANDLE hSession, CK_OBJECT_HANDLE hObject,
+                   CK_ATTRIBUTE_PTR pTemplate, CK_ULONG ulCount,
+                   CK_OBJECT_HANDLE_PTR phNewObject)
+{
+    if (hObject != PRIVATE_KEY_HANDLE && hObject != CERTIFICATE_HANDLE) {
+        return CKR_OBJECT_HANDLE_INVALID;
+    }
+    /* Kludge to make NSS happy */
+    if (pTemplate && ulCount > 0) {
+        return CKR_ACTION_PROHIBITED;
+    }
+    *phNewObject = hObject;
+    return CKR_OK;
```

Comment:
From the pkcs11 spec, it should be valid to always return CKR_ACTION_PROHIBITED on this method if we update attributes.cpp to always return CK_FALSE for the CKA_COPYABLE attribute. This not-really-copying implementation makes me a little bit nervous, so can you test if the above would that make NSS happy?

Reply:
Ah, cool, having CKA_TOKEN=TRUE is both technically correct from my reading of the semantics and saves us from having to decide what to do with C_CopyObject(). Thanks for spending a bunch of time in the debugger. :)